docs/extras/modules/chains/how_to/openai_functions.ipynb

```diff
@@ -342,7 +342,7 @@
     "### Using Python functions\n",
     "We can pass in functions as Pydantic classes, directly as OpenAI function dicts, or Python functions. To pass Python function in directly, we'll want to make sure our parameters have type hints, we have a docstring, and we use [Google Python style docstrings](https://google.github.io/styleguide/pyguide.html#doc-function-args) to describe the parameters.\n",
     "\n",
-    "**NOTE**: To use Python functions, make sure the function arguments are of primitive types (str, float, int, bool) or that they are Pydantic objects."
+    "**Note**: To use Python functions, make sure the function arguments are of primitive types (str, float, int, bool) or that they are Pydantic objects."
```

Comment:
We don't have rules for `note` formatting. All capital is OK, it catches attention well. Just saying.